src/modm/driver/temperature/ds18b20.hpp

```diff
@@ -17,15 +18,25 @@
 
 namespace modm
 {
+	struct ds18b20
```

Comment:
```suggestion
	/// \ingroup	modm_driver_ds18b20
	struct ds18b20
```

Reply:
Fixed. I have also added documentation for the enum and added the `configure()` method to the existing example that it gets instantiated when compiling the driver in the CI.